util/test.sh

```diff
@@ -94,7 +94,20 @@ if [[ "$CI" == 'true' ]]; then
     export NGX_BUILD_FORCE=1
 fi
 
-$NGX_WASM_DIR/util/build_proxy_wasm_go_sdk.sh
+if [[ $PARAMS =~ "t/03-proxy_wasm " \
+      || $PARAMS =~ "t/03-proxy_wasm/sdks " ]];
+then
+    $NGX_WASM_DIR/util/sdk.sh -S assemblyscript --build
+    $NGX_WASM_DIR/util/sdk.sh -S go --build
+fi
+
+if [[ $PARAMS =~ "assemblyscript_sdk" ]]; then
+    $NGX_WASM_DIR/util/sdk.sh -S assemblyscript --build
+fi
+
+if [[ $PARAMS =~ "go_sdk" ]]; then
+    $NGX_WASM_DIR/util/sdk.sh -S go --build
+fi
```

Comment:
I recall we also discussed this actually needed to move _out_ of `test.sh`; only `setup_dev.sh` has the "right" to install new SDKs (if the underlying dependencies are met). The tests just rely on `skip_***` to skip a non-available SDK.